TodoMate/Todo.css

```
@@ -0,0 +1,160 @@
+@font-face {
+  font-family: "BMEULJIRO";
+  src: url("https://cdn.jsdelivr.net/gh/projectnoonnu/noonfonts_twelve@1.0/BMEULJIRO.woff")
+    format("woff");
+  font-weight: normal;
+  font-style: normal;
+}
+
+body {
+  display: grid;
+  place-items: center;
+  font-family: "BMEULJIRO";
+  justify-content: center;
+  text-align: center;
+  background-color: #dbe5f1;
+}
+
+header {
+  font-size: 25px;
+  font-weight: bold;
+  color: white;
+  text-shadow: 0 3px #4699b7, 3px 0 #4699b7;
+}
+
+.calendar {
+  display: grid;
+  grid-template-columns: repeat(7, 1fr);
+  place-items: center;
+  width: 90%;
+  background-color: #bed1a9;
+  border-radius: 30px;
+  box-shadow: 0px 7px 7px #faf3d7;
+  color: #faf3d7;
+  margin-top: -15px;
+  margin-bottom: 30px;
+}
+
+.calendar article {
+  place-items: center;
+  font-size: 20px;
+  line-height: 0.7;
+}
+
+.calendar ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, 1fr);
+}
+
+.calendar li {
+  margin: 15px 0;
+  justify-content: center;
+  align-self: center;
+}
+
+.num {
+  z-index: 1;
+  position: relative;
+  color: #bed1a9;
+  font-size: 20px;
+}
+
+.num:before {
+  color: #faf3d7;
+  content: "♥";
+  margin-top: -3px;
+  margin-left: -15px;
+  font-size: 40px;
+  z-index: -1;
+  position: absolute;
+}
+
+#today {
+  color: #faf3d7;
+}
+
+.plan {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 5px;
+  width: 80%;
+}
+
+.plan article {
+  margin-bottom: 30px;
+  margin-left: -10px;
+  margin-right: 20px;
```

요기도 한줄로도 쓸 수 있습니당!! ㅎㅎㅎ

```suggestion
  margin : 0 20px 30px -10px;
```

---

오 margin이나 padding을 4개를 다 따로 지정하고 싶으면 위에서부터 시계방향 순서로 지정해주면 되는구나..
덕분에 알아갑니당~